mslib/mscolab/models.py

```diff
@@ -222,3 +222,24 @@ def __init__(self, op_id, u_id, commit_hash, version_name=None, comment=None):
             self.version_name = str(version_name)
         if comment is not None:
             self.comment = str(comment)
+
+
+class ViewSettings(db.Model):
+
+    __tablename__ = "viewsettings"
+    id = db.Column(db.Integer, primary_key=True, autoincrement=True)  # noqa: A003
+    op_id = db.Column(db.Integer, db.ForeignKey('operations.id'))
+    u_id = db.Column(db.Integer, db.ForeignKey('users.id'))
```

Comment:
IMO (op_id, u_id) should build a composite primary key, instead of having an additional third id. Then the unique constraint on (op_id, u_id) would be automatic.